test/README.md

```diff
@@ -49,6 +49,28 @@ You can run any combination (incl. duplicates) of tests by calling:
 test/functional/test_runner.py <testname1> <testname2> <testname3> ...
 ```
 
+Wildcard test names can be passed if the paths are coherent.
```

Comment:
Could mention that this requires calling it from a `bash` or similar, since it requires the shell to do the globbing

Reply:
Good idea. Done.